### DISCUSSION TITLE
Monopoly Go facebook login issue

### DISCUSSION THREAD

**Discussion by jnchin** [PlayCover/PlayCover#1030] [Q&A]
had try to disable System Integrity Protection, it works. But after i enabled it. It is back to not working again. Does it mean i will need to disable everytime when i need to use it?

**Comment by thewizardson** [comment]
happy to see another monopoly go user on here. Still trying to figure out how to get it working!

**Reply by putzhead** [reply]
I am working on this also. Trying to figure out how to create capped dice with this app. If you are able to get it working please share

**Comment by The-web-wolf** [comment]
Any luck with this? Can't login with facebook 
![Splynt SS - MONOPOLYGO 2024-04-26 at 18 51 59](https://github.com/PlayCover/PlayCover/assets/30579182/5059501a-a084-499a-b9e8-b57179338a82)
